### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by GromNaN**
Instead of removing the changelog, maintain it with each PR.

Replace https://github.com/mongodb/laravel-mongodb/pull/2668

### Checklist

- [x] Add tests and ensure they pass
- [x] Add an entry to the CHANGELOG.md file
- [x] Update documentation for new features

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Before submitting a pull request:
 
 ## Run Tests
 
-The full test suite requires PHP cli with mongodb extension, a running MongoDB server and a running MySQL server.
+The full test suite requires PHP cli with mongodb extension, a running MongoDB server.
```
Comment (alcaeus) [inline CONTRIBUTING.md]: Is MySQL still required for testing hybrid relationships? If not, you can drop the second sentence, but the suggestion also improves the wording of the first part.

```suggestion
The full test suite requires PHP cli with mongodb extension and a running MongoDB server. A MySQL server is required to test hybrid relationships.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -69,14 +69,12 @@ If the project maintainer has any additional requirements, you will find them li
 
 - **Add tests!** - Your patch won't be accepted if it doesn't have tests.
 
-- **Document any change in behaviour** - Make sure the documentation is kept up-to-date.
+- **Document any change in behaviour** - Make sure the documentation is kept up-to-date. Update the changelog for new features and bug fixes.
```
Comment (alcaeus) [inline CONTRIBUTING.md]:
```suggestion
- **Document any change in behaviour** - Make sure the documentation is kept up-to-date and update the changelog for new features and bug fixes.

```

File: CONTRIBUTING.md (hunk 1 above)
Comment (GromNaN) [reply CONTRIBUTING.md]: MySQL is not necessary since I switched to sqlite: https://github.com/mongodb/laravel-mongodb/pull/2616

Comment (GromNaN) [reply CONTRIBUTING.md]: I added a note for testing transactions.

**Review by alcaeus — APPROVED** [review]

**Pull request merged**